example/zipkin/main.go

```diff
@@ -46,25 +48,32 @@ func initTracer(url string) {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	batcher := sdktrace.NewBatchSpanProcessor(exporter)
+
+	tp := sdktrace.NewTracerProvider(sdktrace.WithSpanProcessor(batcher))
+	global.SetTracerProvider(tp)
```

Comment:
https://pkg.go.dev/go.opentelemetry.io/otel/#SetTracerProvider

```suggestion
	otel.SetTracerProvider(tp)
```